src/container/srv_container.c

```diff
@@ -1401,6 +1488,39 @@ cont_lookup(struct rdb_tx *tx, const struct cont_svc *svc, const uuid_t uuid,
 	return rc;
 }
 
+static int
+cont_lookup_bylabel(struct rdb_tx *tx, const struct cont_svc *svc,
+		    const char *label, struct cont **cont)
+{
+	size_t		label_len;
+	uuid_t		uuid;
+	d_iov_t		key;
+	d_iov_t		val;
+	int		rc;
+
+	label_len = strnlen(label, DAOS_PROP_LABEL_MAX_LEN+1);
+	if (!label || (label_len == 0) || (label_len > DAOS_PROP_LABEL_MAX_LEN))
+		return -DER_INVAL;
+
+	/* Look up container UUID by label */
+	uuid_clear(uuid);
```

Comment:
[Minor] Don't need to zero the uuid buffer.

Reply:
Done in most recent update to the patch.